go/vt/vtgate/engine/set.go

```diff
@@ -472,11 +472,14 @@ func (svss *SysVarSetAware) Execute(ctx context.Context, vcursor VCursor, env *e
 		if err != nil {
 			return err
 		}
-		out, ok := isolationLevelToExecuteOptionIsolationLevel[strings.ToLower(str)]
+		upperStr := strings.ToUpper(str)
```

Comment:
curious to why you switched from lower to upper. not important, just curious :)

Reply:
mysql returns in upper case, so how we store, we return that after rewrite.